samples/SentimentAnalyzerCustomSkill/README.md

```diff
@@ -36,10 +37,12 @@ However, some consumers may also want to tweak and optimize part of that pipelin
 - [Acquiring the sentiment analysis ONNX model](https://github.com/onnx/models/tree/master/emotion_ferplus)
 - [C++/Winrt reference documentation](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/)
 - [MIDL 3.0 syntax reference guide for authoring .idl files](https://docs.microsoft.com/en-us/uwp/midl-3/intro)
+- [Reg-Free WinRT component activation for Win32 apps and linking dependencies](https://blogs.windows.com/buildingapps/2019/04/30/enhancing-non-packaged-desktop-apps-using-windows-runtime-components)
 
 ## System requirements
 
-**Client:** Windows 10 build 17763 or greater
+**Client:** Windows 10 build 17763 or greater for UWP
+**Client:** Windows 10 build 18267 or greater for Win32
```

Comment:
>18267 [](start = 29, length = 5)

I recommend putting 18362 as the RTM of 19H1 instead of pre-release builds that have vanished by now.

Reply:
I ahve done this in my branch which is now merged into this branch, please use iteration 22

---
In reply to: [285268872](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/23#discussion_r285268872) [](ancestors = 285268872)